File: Exercices/ExoJava/exo1.java
public class exo1 {
    public static void main(String[] args) {
        int x = Integer.parseInt(args[0]);
        int y = Integer.parseInt(args[1]);
        int result = sommeDeDeuxNombres(x, y);
        System.out.println("La somme de " + x + " et " + y + " est " + result);
    }

    public static int sommeDeDeuxNombres(int x, int y) {
        return x + y;
    }
}

File: Exercices/ExoC/Exo1C/exo1.c
#include <stdio.h>

int sommeDeDeuxNombres(int x, int y) {
    return x + y;
}

int main() {
    printf("%d %d %d", sommeDeDeuxNombres(2, 2), sommeDeDeuxNombres(7, 3), sommeDeDeuxNombres(1, 6));
    return 0;
}

